### ISSUE TITLE
wrong folder after last update (3.0.2)

### ISSUE THREAD

**Issue by carlosCastro99** [rospogrigio/localtuya#99]
it seems that the last update has a bug. it's writing files in directory /config/custom_components/localtuya/custom_components/localtuya.
Had to move them manually.

**Comment by rospogrigio** [comment]
Yeah sorry guys, I made some mess, please downgrade back to 3.0.1 and wait for localtuya to be added into HACS natively.
Thank you

**Comment by sabellwind** [comment]
Hi guys, firstly, thanks for getting this going. It's looking very promising!

Not sure if this is related, but I downgraded, and now my device is not showing up anymore. It is not discovering automatically, and when added manually and via yaml, the following error is thrown:
```
Connect to 192.168.1.134 failed
Traceback (most recent call last):
  File "/config/custom_components/localtuya/common.py", line 151, in _make_connection
    status = await self._interface.status()
  File "/config/custom_components/localtuya/pytuya/__init__.py", line 429, in status
    if "dps" in status:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by postlund** [comment]
@sabellwind I have seen that issue as well, I think the device decides to close the connection. I will commit a workaround for it.

**Comment by sabellwind** [comment]
Hi @postlund I managed to trace the issue in this instance to TuyaGateway running at the same time. I was experimenting with it but gave up after I came across this more simply put together integration. It was configured with the device ID, virtual key and IP, but not much more than that.

Seems as if adding the device won't work if it's actively being engaged by another platform, (tuya, TuyaGateway etc.) 

Thanks for following up
